tools/gosec/gosec.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"encoding/xml"
+	"flag"
+	"io/ioutil"
+	"log"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+// These structs are copied from gosec
+type junitXMLReport struct {
+	XMLName    xml.Name    `xml:"testsuites"`
+	Testsuites []testsuite `xml:"testsuite"`
+}
+
+type testsuite struct {
+	XMLName   xml.Name   `xml:"testsuite"`
+	Name      string     `xml:"name,attr"`
+	Tests     int        `xml:"tests,attr"`
+	Testcases []testcase `xml:"testcase"`
+}
+
+type testcase struct {
+	XMLName xml.Name `xml:"testcase"`
+	Name    string   `xml:"name,attr"`
+	Failure failure  `xml:"failure"`
+}
+
+type failure struct {
+	XMLName xml.Name `xml:"failure"`
+	Message string   `xml:"message,attr"`
+	Text    string   `xml:",innerxml"`
+}
+
+var (
+	junitFile  string
+	outputFile string
+)
+
+func init() {
+	flag.StringVar(&junitFile, "junit", "junit-gosec.xml", "usage")
+	flag.StringVar(&outputFile, "output", "", "usage")
+}
+
+func parseJunit(filename string) *junitXMLReport {
```

Comment:
For improved readability the called methods should either appear all below the calling code or all above, at best in order of calls.